docs/user_guide/2_connecting_the_cables.rst

```diff
@@ -0,0 +1,36 @@
+Connecting Cables
+------------------
+
+* Connect the hands to the NUC-CONTROL. It is very important that the exact USB-ethernet adapters are used.
+
+  * The right hand should be connected to a USB->ethernet adapter labelled: ``HAND RIGHT``, which should be connected to one of the USB ports of the NUC-CONTROL (it does not matter which one).
+  * The left hand should be connected to a USB->ethernet adapter labelled: ``HAND LEFT``, which should be connected to one of the USB ports of the NUC-CONTROL (it does not matter which one).
+
+* Connect one USB->Ethernet adapter labelled ``NUC-CONTROL`` to another USB port on the NUC and the other USB->Ethernet labelled ``SERVER`` to any of the ports in your ``SERVER`` Laptop (provided by Shadow or a custom one).
+* Connect the two adaptors together with an Ethernet cable.
+
+You have been supplied with medium length Ethernet leads, but if you require a different length, you can simply use a standard commercial Ethernet Cat 5 cable, available from most computer parts suppliers. If you require internet connection in the laptop, connect an ethernet cable providing external internet connection to the back of the laptop, to an ethernet port labelled ``INTERNET``.
+
+.. figure:: ../img/connecting_the_bimanual_hand.png
+    :width: 100%
+    :align: center
+    :alt: Connections diagram
+
+    Connections diagram
+
+.. Source to edit the diagram: https://docs.google.com/drawings/d/1IOYFVruiCEKmIZpWwnUS8AJ-SWSNJJGQQxWrUoBa2Hk/edit?usp=sharing
+
+* Finally, connect the external power supply to the hands using the metal Lemo connector, making sure to line up the red dots. When power is applied to the hand, the fans will be heard immediately. If you require a longer or shorter cable, please contact us at support@shadowrobot.com. 
+
+Connection procedure
+^^^^^^^^^^^^^^^^^^^^^
+1. Connect the ethernet between the NUC and the laptop using the instructions above
+2. Power on the laptop
+3. Connect an ethernet cable providing external internet connection to the back of the laptop
+4. Power on the NUC
+5. Make sure the laptop has only 1 USB-Ethernet adapter connected to it.
+6. In case of using another laptop than one provided, please follow the instructions below to install the software.
+7. Power on the hand(s)
+8. Connect the right hand to the USB-ethernet adapter labelled “HAND RIGHT” which should be plugged in to the NUC, as explained above
```

Comment:
Above you use `USB->ethernet` but here `USB-ethernet`. Should it be the same everywhere?

Reply:
fixed